app/frontend/components/ConcertoVimeoVideo.vue

```diff
@@ -28,6 +29,20 @@ function isVimeoAPILoaded() {
 
 async function loadVimeoAPI() {
   return new Promise((resolve) => {
+    // Check if script is already in the DOM
+    const existingScript = document.querySelector('script[src="https://player.vimeo.com/api/player.js"]');
```

Comment:
I see the `https://player.vimeo.com/api/player.js` string is also used below to construct the script tag.  Can we DRY this up so we're not maintaining two version of it?